Scenarios/AKS-Secure-Baseline-PrivateCluster/Bicep/02-AAD/ad_groups.ps1

```diff
@@ -16,5 +16,5 @@ if($isInstalled){
     New-AzADGroup -DisplayName $aksops -MailNickname $aksops
 }
 else {
-    Write-Output "Azuer Powershell not installed. Installation steps in: https://docs.microsoft.com/en-us/powershell/azure/install-az-ps"
+    Write-Output "Azuer Powershell not installed. Installation steps in: https://learn.microsoft.com/powershell/azure/install-az-ps"
```

Comment:
Azure is typo'd here. May as well fix :)

Reply:
```suggestion
    Write-Output "Azure PowerShell not installed. Installation steps in: https://learn.microsoft.com/powershell/azure/install-az-ps"
```